<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settng_Distance</class>
 <widget class="QWidget" name="Settng_Distance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:black;
}</string>
  </property>
  <widget class="QLabel" name="LBL_ShuttleDis">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>132</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(255,255,255);
color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TotalDis">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(255,255,255);
color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_MSCount">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(255,255,255);
color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Unit3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>次</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Unit1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>132</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>Km</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Unit2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>Km</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_tip3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font:14px;</string>
   </property>
   <property name="text">
    <string>主断闭合次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_tip1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>112</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font:14px;</string>
   </property>
   <property name="text">
    <string>往返里程</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_tip2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font:14px;</string>
   </property>
   <property name="text">
    <string>总里程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Reset">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>132</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(255,255,255);
background-color:rgb(0,0,0);
color:rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>重设</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
